### DISCUSSION TITLE
Ticwatch pro 5 standalone collector

### DISCUSSION THREAD

**Discussion by turtlepaw123** [NightscoutFoundation/xDrip#3266] [General]
So I had a ticwatch pro 3 ultra, wear os 3.5, that would work as a standalone collector whenever I manually forgot the dexcom transmitter as a saved device. Worked nicely for when I didn't have access to my phone. Not exactly practical.

I just upgraded to a Ticwatch pro 5 and no matter the settings configuration I use, it won't work. (I found scanning always on was crucial for the old watch. Other settings wouldn't matter.)

Is there anything I can do or anywhere to better ask for help? I don't fully get the tech stuff. It's the same wear os, same company, android 11. Don't get why it wouldn't work similar enough.

Thanks for any comments.

**Comment by turtlepaw123** [comment]
Forgot to add, for standalone collecting use with a Dexcom g6, through xDrip. It won't even attempt to connect with the transmitter. No matter settings enabled/disabled. Scanning, unbond, screen on scan, etc

**Reply by Unai-HP** [reply]
You were able to get the collector to work on Wear3???? I've been (and a lot more people) trying to use my Galaxy Watch 4 as a collector and It has never worked, and seems that the wearos code for collection is very incomplete. Are you sure it worked???

**Reply by turtlepaw123** [reply]
It may have been when the previous ticwatch was on wear 2.something now that I think...sorry. unsure. Will check later and update.
